timeout/timeout_test.go

```diff
@@ -42,7 +43,7 @@ func (s *TimeoutTestSuite) TestTimeoutMarshal(c *C) {
 }
 
 type testT struct {
-	T Timeout
+	T Timeout `yaml:"T"`
```

Comment:
I suppose your comment was about needing this yaml tag? yes, that seems odd and different from encoding/json